AABB_tree/package_info/AABB_tree/dependencies

```diff
@@ -0,0 +1,19 @@
+ 
```

Comment:
Any reason to have an empty line in all the files?

Reply:
Yes, it is because we create the file empty, it creates an empty line. I guess it is to ensure that the file exists even if there is no dependency.